src/desktop/lib/global_client_setup.tsx

```diff
@@ -2,9 +2,17 @@
  * Set webpack public-path asset lookup to CDN in production
  */
 if (process.env.NODE_ENV === "production") {
-  __webpack_public_path__ = process.env.CDN_URL + "/assets/"
+  __webpack_public_path__ = "https://d1rmpw1xlv9rxa.cloudfront.net/assets/"
```

Comment:
Need to fix this hard-code, but not sure why `process.env.CDN_URL` isn't being passed through at compile time, even though its set in webpack `baseConfig`.

Reply:
Does `CDN_URL` have to be available when `yarn assets` is run for this to work? That's in CI rather than the kubernetes environment.

Reply:
Ah interesting, that's probably the issue. I assumed whatever drove `hokusai staging env get` env wise was piped into CI.